terraform/application/application.tf

```diff
@@ -79,28 +79,3 @@ module "worker_application" {
 
   enable_logit = var.enable_logit
 }
-
-module "migration_worker_application" {
```

Comment:
should we also change the other worker to pick up all queues now (remove specifying the queue on the other worker)?

Reply:
Ah, yeah good catch I'll update it